react/ReadMore/ReadMore.less

```diff
@@ -0,0 +1,20 @@
+@import (reference) "~seek-style-guide/theme";
+
+.content {
+  position: relative;
+  display: block;
+  word-break: break-word;
+  white-space: pre-wrap;
+}
+
+.fadeOut {
+  background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 0%, white 100%);
+  position: absolute;
+  height: (@row-height * 4);
+  width: 100%;
+  bottom: 0;
+}
+
+.chevron {
+  margin-left: 5px;
```

Comment:
Don't want to block this now, but we should figure out a way to remove the need for custom horizontal icon spacing.